go.mod

```diff
@@ -31,6 +31,7 @@ require (
 	github.com/mattn/go-isatty v0.0.20
 	github.com/oapi-codegen/oapi-codegen/v2 v2.4.1
 	github.com/oapi-codegen/runtime v1.1.1
+	github.com/oklog/ulid v1.3.1
```

Comment:
Question: Is this dependency important for the functionality? Can't we just treat the token as a string?